eventkit_cloud/utils/tests/test_pcf.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import logging
+from django.test import TestCase
+from ..pcf import PcfClient
+import json
+import requests_mock
+
+logger = logging.getLogger(__name__)
+
+
+class TestPcfClient(TestCase):
+
+    def setUp(self):
+        self.mock_requests = requests_mock.Mocker()
+        self.mock_requests.start()
+        self.addCleanup(self.mock_requests.stop)
+
+        self.url = 'http://example.dev'
+        self.org = 'org'
+        self.space = 'space'
+
+
+    def test_get_providers(self):
```

Comment:
This test appears to be some copy and paste, from the eventkit client, can you update these test cases for the PCF client?

Reply:
Added tests for each PcfClient() function.